x-pack/solutions/security/plugins/security_solution/public/entity_analytics/components/privileged_user_monitoring/components/key_insights_panel/index.test.tsx

```diff
@@ -0,0 +1,250 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { render } from '@testing-library/react';
+import { TestProviders } from '../../../../../common/mock';
+import { KeyInsightsPanel } from '.';
+import type { DataViewSpec } from '@kbn/data-views-plugin/public';
+
+// Mock all tile components
+jest.mock('./active_privileged_users_tile', () => ({
```

Comment:
Same comment here, too. We should avoid mocking components as much as possible to make test execution as realistic as possible.